sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-retraining-pipeline.ipynb

```diff
@@ -0,0 +1,440 @@
+    "                image_filename = root.find(\"filename\").text\n",
+    "                _, file_extension = os.path.splitext(image_filename)\n",
+    "                json_line = dict(json_line_sample)\n",
+    "                json_line[\"image_url\"] = (\n",
+    "                    json_line[\"image_url\"] + \"images/\" + image_filename\n",
+    "                )\n",
+    "                json_line[\"image_details\"][\"format\"] = file_extension[1:]\n",
+    "                json_line[\"image_details\"][\"width\"] = width\n",
+    "                json_line[\"image_details\"][\"height\"] = height\n",
+    "                json_line[\"label\"] = labels\n",
+    "\n",
+    "                if i % train_validation_ratio == 0:\n",
+    "                    # validation annotation\n",
+    "                    validation_f.write(json.dumps(json_line) + \"\\n\")\n",
+    "                else:\n",
+    "                    # train annotation\n",
+    "                    train_f.write(json.dumps(json_line) + \"\\n\")\n",
+    "            else:\n",
+    "                print(\"Skipping unknown file: {}\".format(filename))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Retraining pipeline job with Image Object Detection task\n",
+    "\n",
+    "## 2.1 Load model register component"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import load_component\n",
+    "parent_dir = \".\"\n",
+    "register_model = load_component(path=parent_dir + \"/register_model_pipeline.yml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.2 Build retraining pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# note that the used docker image doesn't suit for all size of gpu compute. Please use the following command to create gpu compute if experiment failed\n",
+    "# !az ml compute create -n gpu-cluster --type amlcompute --min-instances 0 --max-instances 4 --size Standard_NC12"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define pipeline\n",
+    "@dsl.pipeline(\n",
+    "    name=\"AutoML Image Object Detection Retraining Pipeline\", default_compute_target=\"gpu-cluster\"\n",
+    "    )\n",
+    "def automl_image_object_detection_retraining(\n",
+    "    image_object_detection_train_data,\n",
+    "    image_object_detection_validation_data\n",
+    "):\n",
+    "    # define the automl image_object-detection task with automl function\n",
+    "    image_object_detection_node = image_object_detection(\n",
+    "        training_data=image_object_detection_train_data,\n",
+    "        validation_data=image_object_detection_validation_data,\n",
+    "        target_column_name=\"label\",\n",
```

Comment:
do we need to pass target_column_name and primary_metric? we are not using the defaults if we didn't?